### DISCUSSION TITLE
Difficulty Connecting Artisan to Typhoon 2.5kg Electric Roaster

### DISCUSSION THREAD

**Discussion by gnosiscoffeeroasters** [artisan-roaster-scope/artisan#1494] [Q&A]
I have followed the steps given by Typhoon Roasters, but still cannot successfully connect to Artisan.
Here is what I have...
1) "Modbus Error: failed to connect" 
2) Latest version of Artisan- 2.10.2
3) IP Adress: 192.168.0.100
4) Mask: 255.255.255.0
5) Gateway: 8.8.8.8
6) DNS: 8.8.8.8
Any ideas???

**Comment by poundy** [comment]
from the doco, https://artisan-scope.org/machines/typhoon/, it appears you have set up your network correctly. Can you confirm that your roaster has the default IP address of 192.168.0.210 ? Can you communicate to it (perhaps using PING) to see if it's responding? 

Failed to connect means it can't communicate to the roaster, either the roaster is not on that IP address, your PC is not able to reach that network IP address, or for some other reason the device is not responding. Perhaps your machine is newer and does not have that same IP address? Have you checked with the doc you got with the machine, or with Typhoon themselves?

**Comment by SummerLad** [comment]
Did you manage to solve the issue @gnosiscoffeeroasters ? I´m facing the same problem with this model. 

Another isseu I´m having is that my computer only identifies the machine when it is on Prep mode. Typhoon´s technical support acknowledges it isn't right, but has no clue on the reason it´s happening.

**Reply by MAKOMO** [reply]
Your comment is very machine specific. Who knows was "Prep mode" is and what you exactly mean by "identifies the machine". More details are always welcome.

However, as long as your computer cannot talk to your machine, Artisan will not be be able to talk to it. Thus we are working on an AI-based telepathic mode to reach out to all machines world-wide via broad cosmic waves....

**Reply by SummerLad** [reply]
Yes... the comment was very machine specific, because my problem is very machine specific too. I asked it to @gnosiscoffeeroasters exactly because he has the same machine and probably already found a solution for the same problem.

Nevertheless, I´m looking forward to the AI Telepathic Connection Mode. It would definitely make my life much easier now ;)

**Reply by MAKOMO** [reply]
The problem is that nobody here knows this machine and its specific thus it will get hard to support you. Typhoon should be able to help you out.
